src/libstd/io/stdio.rs

```diff
@@ -204,6 +204,30 @@ impl Stdin {
     ///
     /// For detailed semantics of this method, see the documentation on
     /// `BufRead::read_line`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use std::io;
+    ///
+    /// #fn main() {
+    ///     let mut input = String::new();
+    ///     match io::stdin().read_line(&mut input) {
+    ///         Ok(n)      => {
+    ///             println!("{} characters read", n);
```

Comment:
I believe that you'll also need to import `std::io::prelude::*` to get this example to work. (you can be sure to run this test via `make check-stage2-doc-crate-std`).